ASP.NET Core/Data/AspNetCoreTemplate.Data/Repositories/EfRepository.cs

```diff
@@ -25,7 +25,7 @@ public EfRepository(ApplicationDbContext context)
 
         public virtual IQueryable<TEntity> AllAsNoTracking() => this.DbSet.AsNoTracking();
 
-        public virtual Task AddAsync(TEntity entity) => this.DbSet.AddAsync(entity);
+        public virtual async Task AddAsync(TEntity entity) => await this.DbSet.AddAsync(entity);
```

Comment:
Can use `ToTask()` to convert the `ValueTask` to `Task`.

Reply:
Fixed.